packages/protocol/src/data/form.ts

```diff
@@ -15,14 +16,57 @@ export type OrientationType = "horizontal" | "vertical";
 export type TextIconStyle = "INLINE" | "BLOCK";
 export type TextType = "RAW" | "MARKDOWN";
 
+export interface Forms {
+  cmsMetaRequest: CmsMetaRequest;
+  compositeInfo: CompositeInfo[];
+  contentObject: ContentObject[];
+  form: Form;
+  formContext: FormContext;
```

Comment:
I now renamed the 'wrapper' rather than the concrete type 'Form' ... also ok I think? at least much easier to achieve :wink: